tests/raftstore/transport_simulate.rs

```diff
@@ -637,3 +637,25 @@ impl Clone for RandomLatencyFilter {
         }
     }
 }
+
+#[derive(Clone, Default)]
+pub struct LeaseReadFilter {
```

Comment:
I prefer keeping these in the original places. They are only used in one case, keep them close makes it easier to read the case.

Reply:
I am going to use it in #2639 .